Dockerfile

```diff
@@ -6,27 +6,20 @@ RUN cd /tmp && \
     wget https://dl.google.com/go/go1.12.7.linux-amd64.tar.gz && \
     tar -xf go1.12.7.linux-amd64.tar.gz && \
     mv go /usr/local
-WORKDIR /go/src/github.com/prebid/prebid-server/
+RUN mkdir -p /app/prebid-server/
+WORKDIR /app/prebid-server/
 ENV GOROOT=/usr/local/go
-ENV GOPATH=/go
-ENV PATH=$GOPATH/bin:$GOROOT/bin:$PATH
-RUN apt-get update && \
-    apt-get install -y git go-dep && \
-    apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+ENV PATH=$GOROOT/bin:$PATH
 ENV CGO_ENABLED 0
 COPY ./ ./
-RUN dep ensure && \
-    go build .
+RUN go build -mod=vendor .
```

Comment:
Why do you recommend using the vendor directory versus the default $GOPATH\pkg\mod directory?

Reply:
No preferences, but IMO modules allowing to work outside of the GOPATH is one of the best features, so just wanted to take this opportunity :)

Reply:
I am not opposed :), was just curious if you ran into an issue specific to building the Docker container. I like that it allows for a more targeted cache clean by the deps command.

Reply:
So it's ok for you this way? Also, I might introduce in another PR later a GOPROXY for modules to speed up the build.

Reply:
Yes. I don't see a problem with it. 

The proxy sounds interesting. I'm curious to see what that looks like.

Reply:
Hey @SyntaxNode , here you go for the proxy: #1079 :)